src/index.html

```diff
@@ -11,6 +11,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+    >
+
+      <div class="search-field">
+
+        <img
+          src="images/Search.svg"
+          class="search-icon-big"
+          alt="Search icon"
+        >
+        <input
+          type="search"
+          class="search-bar big-search-bar"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+      </div>
+
+    </form>
```

Comment:
Please, fix the code style everywhere
```suggestion
      </div>
    </form>
```